src/main/resources/services/asset/requestHandler.ts

```diff
@@ -147,6 +152,46 @@ export function requestHandler({
       });
     }
 
+    if (doStaticCompression() && request.headers?.['accept-encoding']) {
+      let compression: AcceptEncodingCompressionFormat | undefined;
+      let highestWeight = 0;
+      request.headers['accept-encoding'].split(/\s*,\s*/).forEach((encoding: AcceptEncodingItem) => {
```

Comment:
Weight is really advanced. Browsers don't use weights. They rely on order.
And worse, you don't test for ;q=0
I suggested in spec that `br` is preferred, because. Just like we do in booster https://github.com/enonic/app-booster/blob/master/src/main/java/com/enonic/app/booster/servlet/RequestUtils.java#L69

Reply:
> And worse, you don't test for ;q=0

Yeah I didn't read the whole thing:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Encoding

This text is confusing though (double negative):

> As long as the identity;q=0 or *;q=0 directives do not explicitly forbid the identity value that means no encoding, the server must never return a [406](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/406) Not Acceptable error.

Reply:
So if the Accept-Encoding header contains`identity;q=0` or `*;q=0` we should return 406

Reply:
Did not do the 406, but ;q=0 should work now.